product-web-app/views/base/navbar.html

```diff
@@ -18,9 +18,11 @@
     {{title}}
   </button>
   <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-    {% if role === "Admin" %} {% endif %}
-    <a class="dropdown-item" href="/job-roles">Job Roles</a>
+    {% if role == "Admin" %} 
+    {% endif %}
```

Comment:
is this a merge mistake? Admin should see all bellow pages or at least have only admin pages in that if (/admin/add-band)